### DISCUSSION TITLE
Enabling sigma alerts (via playbook) over keyword type

### DISCUSSION THREAD

**Discussion by idolaman** [Security-Onion-Solutions/securityonion#4556] [Unsupported Versions] [answered]
Hello,
We trying to create alert with playbook GUI, the alert trying resolve exact match with command and path.
Unfortunately, it is not possible because the automatic adding of .security analyzer, when using the 'Create new play' feature the alert created with .security in text fields (the problem is the .security analyzer in es tokenize the data by specific separators and the data cannot be queried with exact match)

There is simple way to remove the .security when creating new play in  playbook GUI?

Thank you very much!

**Comment by defensivedepth** [comment, marked as answer]
There is not an easy way to override this at the moment, primarily because the `.security` analyzer is definitely the best way to go because of case insensitive wildcard searches. 

Instead, we can get the analyzer fixed for the issue you are seeing - can you please post details?

Also, what version of SO are you on?
